compio-quic/src/endpoint.rs

```diff
@@ -310,10 +311,83 @@ impl EndpointInner {
     }
 }
 
+#[derive(Debug, Clone)]
+pub(crate) struct EndpointRef(Arc<EndpointInner>);
+
+impl EndpointRef {
+    // Modified from [`SharedFd::try_unwrap_inner`], see notes there.
+    unsafe fn try_unwrap_inner(this: &ManuallyDrop<Self>) -> Option<EndpointInner> {
+        let ptr = ManuallyDrop::new(unsafe { std::ptr::read(&this.0) });
```

Comment:
This ManuallyDrop is not needed.

Reply:
Oh, you removed another `ManuallyDrop`. LOL.

Reply:
> It has already been simplified.

> Oh, you removed another `ManuallyDrop`. LOL.

Yeah I mean "that" `ManuallyDrop`. :joy_cat: